docs/topics/modelling_considerations.md

```diff
@@ -0,0 +1,194 @@
+
+There are some considerations to make when designing how to model a project in the platform. This document will give you
+some hints to avoid most common mistakes (use them as hints to guide your modelling, not as strict rules).
+
+## The IoT Platform is centered in context
+
+The central piece of Telefonica's IoT Platform is the Context Broker: a component that lets you store and query information
+about context entities. But, what is a Context Entity? Context Entities are logical abstractions that represent the elements
+of your system. Some examples are:
+
+* If you are modelling a Smart City, you may have entities like: Streetlight, Garden, Bus, Street, RecycleBin...
+* Modelling a Industry you may find useful entities like: Worker, Machine, Product, and so on.
+* When designing a Smart Parking you will have entities like: Car, User, Floor...
+
+What does it mean that those objects are Context Entities (instead of classes, objects, tables, or documents). The
+difference lays in how it is treated around the platform: modifications of the CEs can be observed by subscriptions, the
+history of those changes stored, rules can be defined to modify the global Context based on CEs changes.
+
+All the given examples have several things in common:
+
+* All of them model nouns: Context Entities are subjects of interactions; they have a history, they may have subscriptions
+to their changes, etc. This behavior does not fit well with processes (and things typically modelled as verbs or actions).
+
+* All those entities are expected to have a long life in the system. Entities are not supposed to model
+transient information (e.g.: support tickets, mails, log files). This doesn't mean short-lived entities can't be modelled
+in the system, but having those kind of entities in your project will heavily impact the performance
+
+## The Platform is not a ticketing system
+
+Another common mistake involves the use of the Platform for short-lived ticketing, as in the following scenarios:
+
+* A system that keeps track of alarms risen by sensors, and whether they have been solved or not. Modelled entities are:
+Sensor and Alarm.
+
+* A booking system for Theater seats, modelling each ticket as a Context Entity. Modelled entities are: Seat and Ticket.
+
+In both cases there is a Context Entity that does not comply with the above rules about Context Entities: Ticket and Alarm.
+Both entities model short lived elements that will be created once and again in the system, cluttering your service with
+old information.
+
+Objects like tickets, even being nouns, model interactions between entities in your system instead of modelling contextual
+elements: a ticket is a temporal association between a user (that may go once and again to the same theater) and the seat
+(where he will be seated in a particular play). A more convenient way to model this kind of information would be to
+directly code it in the seat, as in the following example:
+
+      {
```

Comment:
Same comment about NGSIv1 convencience. In this case maybe is more feasible to directly use NGSIv2 (as this section is not so "tied" to IOTAs).